README.md

```diff
@@ -34,9 +34,12 @@ eventra.emit('greeting', "Hello User");
 
 # JavaScript Implementation
 To implement this into your JavaScript application, it is a very similar process to the one listed above.  Simply declare your event emitter and start using it.
+You will need the eventra package from `@duxcore/eventra` which can be installed using 
+npm: `npm install @duxcore/eventra` or through yarn: `yarn add @duxcore/eventra`
```

Comment:
Not sure this is needed. I'd probably just add an installation heading with these commands.

```suggestion
```